src/scenes/Main/Main.view.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import classNames from 'classnames';
+
+import { Title } from '@game/components';
+import * as Game from './components';
+
+import { IOptions } from '@game/types';
+
+import './style.scss';
+
+interface IMain {
+    options: IOptions;
+    winner: boolean;
+    turn: boolean;
```

Comment:
i thinks better to use interface for type instead of turn boolean if it's not a game with machine

Reply:
?

Reply:
i mean what's use of turn property? is it like isUserTurn?